src/research.py

```diff
@@ -88,89 +115,311 @@ def get_info(self):
+weapon_list = [tanto, halberd, poleaxe, great_axe, rapier, polehammer, sword, mace, falx, shortsword, bastard_sword, claymore, spear, staff, pike, sarissa, axe, flail, morning_star, bill]
+
+sidearm_list = [unarmed, shield, leather_shield, wooden_shield, buckler, round_shield, kite_shield, heater_shield, board_shield, tower_shield, pavise, pick, hatchet, falchion, sabre, cleaver, sickle, knife, hand_axe, tanto, dagger, club, mace, kopis, hammer, rondel, dirk, staff, shortsword, axe, spear]
+basic_weapon_list = [katana, macuahuitl, falchion, sabre, war_axe, battle_axe, goedendag, short_spear, short_voulge, club, mace, hammer, staff, shortsword, axe, spear]
+
+sling = Weapon('Sling', 15, 250, 0, 3, 1, 1.8, 1, reload_time=60, ammunition=25, projectile_speed=7, armor_pierce=-1, projectile_size=2)
+javelin = Weapon('Javelin', 100, 125, 0.5, 6, 2, 1.5, 1, reload_time=20, ammunition=3, projectile_speed=5, armor_pierce=1, projectile_size=10)
 atlatl = Weapon('Atlatl', 150, 175, 0.5, 8, 1, 2.0, 1, reload_time=70, ammunition=8, projectile_speed=6)
-shortbow = Weapon('Shortbow', 75, 300, 0.5, 4, 1, 2, 1, reload_time=70, ammunition=15, projectile_speed=10)
-bow = Weapon('Bow', 125, 350, 0.5, 5, 1, 2, 1, reload_time=80, ammunition=15, projectile_speed=12)
-longbow = Weapon('Longbow', 200, 400, 0.5, 6, 1, 2.5, 1, reload_time=90, ammunition=15, projectile_speed=14)
-crossbow = Weapon('Crossbow', 250, 450, 1.5, 10, 1, 1, 1, reload_time=200, ammunition=15, projectile_speed=20)
-sling_staff = Weapon('Sling Staff', 60, 300, 0, 5, 2, 2, 1, reload_time=60, ammunition=20, projectile_speed=12)
-
-all_ranged_weapons = [atlatl, sling, shortbow, longbow, javelin, bow, crossbow, sling_staff]
-ranged_weapon_list = [sling, atlatl, javelin, shortbow, longbow, bow, crossbow, sling_staff]
-
-basic_ranged_weapon_list = [sling, javelin, shortbow, bow]
+shortbow = Weapon('Shortbow', 75, 300, 0.5, 4, 1, 2, 1, reload_time=70, ammunition=15, projectile_speed=10, projectile_size=6)
+bow = Weapon('Bow', 125, 350, 0.5, 5, 1, 2, 1, reload_time=80, ammunition=15, projectile_speed=12, projectile_size=8)
+longbow = Weapon('Longbow', 200, 400, 0.5, 6, 1, 2.5, 1, reload_time=90, ammunition=15, projectile_speed=14, armor_pierce=1, projectile_size=9)
+crossbow = Weapon('Crossbow', 250, 450, 1.5, 10, 1, 1, 1, reload_time=120, ammunition=15, projectile_speed=20, armor_pierce=1, projectile_size=4)
+sling_staff = Weapon('Sling Staff', 60, 300, 0, 5, 2, 2, 1, reload_time=60, ammunition=20, projectile_speed=12, armor_pierce=-1, projectile_size=2)
+
+#Ranged - Kenny Additions
+#don't know if handguns will be suited to the time-period but adding anyways
```

Comment:
Handguns are fine, assuming you mean flintlock/matchlock--not going for anything modern here. Probably get rid of this comment though.